sim/midas/src/main/cc/rtlsim/Makefrag-vcs

```diff
@@ -18,7 +18,7 @@
 VCS ?= vcs -full64
 override VCS_FLAGS := -quiet -timescale=1ns/1ps +v2k +rad +vcs+initreg+random +vcs+lic+wait \
 	-notice -line +lint=all,noVCDE,noONGS,noUI -quiet -debug_pp +no_notifier -cpp $(CXX) \
-	-top emul \
+	-top $(TB) \
```

Comment:
Oops.  Looks like I didn't test my change to add `-top` very well.  Thanks for fixing.